BookReaderDemo/demo-internetarchive.html

```diff
@@ -88,19 +88,17 @@ <h3 id="placeholder">please wait as we are fetching the following: </h3>
     <h3>Placeholder div to allow scrolling</h3>
   </div>
 
+  <modal-manager></modal-manager>
```

Comment:
question: if this is a demo, and we're changing things, can there be an <!-- --> explanation here?

Reply:
thanks, added